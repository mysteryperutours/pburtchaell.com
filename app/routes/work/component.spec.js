import React from 'react';
import WorkRoute from './component';

describe(WorkRoute.name, () => {
  it('should render component', () => {
    expect(shallow(<WorkRoute />)).toMatchSnapshot();
  });
});
