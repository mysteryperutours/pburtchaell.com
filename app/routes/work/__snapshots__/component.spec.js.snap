exports[`WorkRoute should render component 1`] = `
<RouteContainer>
  <Row
    defaultColumn={true}
    elementType="div">
    <Text
      style={
        Object {
          "marginBottom": "4rem",
          "marginTop": "1rem",
        }
      }
      type="h1">
      All Work
    </Text>
  </Row>
</RouteContainer>
`;
